edgeml_pytorch/graph/rnn.py

```diff
@@ -332,14 +339,20 @@ def __init__(self, input_size, hidden_size,
                  name="FastRNN"):
         super(FastGRNNCell, self).__init__(input_size, hidden_size,
                                            None, update_nonlinearity,
-                                           1, 1, wRank, uRank, wSparsity, uSparsity)
+                                           1, 1, 1, wRank, uRank, wSparsity,
+                                           uSparsity)
 
         self._alphaInit = alphaInit
         self._betaInit = betaInit
         if wRank is not None:
             self._num_W_matrices += 1
+            self._num_weight_matrices[0] = self._num_W_matrices
         if uRank is not None:
             self._num_U_matrices += 1
+            self._num_weight_matrices[1] = self._num_U_matrices
+        if uRank and wRank:
+            self._num_biases += 1
```

Comment:
@pushkalkatara num_biases is independent on uRank and wRank but rather dependent on the bias parameters which look like self.bias_*. Please update this accordingly. For FastGRNN it will be 2 bias terms, FastRNN has 1, UGRNN as 2, GRU has 3 and LSTM has 4, simple RNN has 1

Reply:
Also please check getVars() function to see what we are trying to do with this num_biases. This counting list is a easy way to access getVars()

Reply:
Yes, I just noticed it. I'll make the changes accordingly.